Source/MultipartFormData.swift

```diff
@@ -453,18 +430,18 @@ public class MultipartFormData {
         return encoded as Data
     }
 
-    private func encodeHeaderDataForBodyPart(_ bodyPart: BodyPart) -> Data {
+    private func encodeHeaderData(for bodyPart: BodyPart) -> Data {
         var headerText = ""
 
         for (key, value) in bodyPart.headers {
-            headerText += "\(key): \(value)\(EncodingCharacters.CRLF)"
+            headerText += "\(key): \(value)\(EncodingCharacters.crlf)"
         }
-        headerText += EncodingCharacters.CRLF
+        headerText += EncodingCharacters.crlf
 
         return headerText.data(using: String.Encoding.utf8, allowLossyConversion: false)!
     }
 
-    private func encodeBodyStreamDataForBodyPart(_ bodyPart: BodyPart) throws -> Data {
+    private func encodeBodyStreamData(for bodyPart: BodyPart) throws -> Data {
```

Comment:
Agreed. I took it a step further and reworked the APIs on most of the `encode` method in 6bcbf5b0.